src/client/subscriptions.rs

```diff
@@ -133,11 +133,11 @@ pub struct Subscription<C = Customer> {
     /// automatically be charged with the saved payment method on the due date.
     ///
     /// If `None`, the value is determined by the plan configuration.
-    pub auto_collection: bool,
```

Comment:
is the problem that the API is sending `None` but the library is failing to deserialize? If so, maybe we should use the serde attribute to indicate that a missing value is deserialized as false

but I might have misunderstood

Reply:
Yeah that's right, the API sends None so I think the library can't deserialize it properly. The Orb API marks this field as both required and nullable, so I think making it an Option instead of a serde attribute to allow it to be missing makes more sense.